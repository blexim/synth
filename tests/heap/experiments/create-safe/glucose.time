WARNING! DIMACS header mismatch: wrong number of variables.
Command exited with non-zero status 20
0.18user 0.00system 0:00.19elapsed 98%CPU (0avgtext+0avgdata 18896maxresident)k
0inputs+16outputs (0major+5550minor)pagefaults 0swaps
