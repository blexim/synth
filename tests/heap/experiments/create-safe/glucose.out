c
c This is glucose 4.0 --  based on MiniSAT (Many thanks to MiniSAT team)
c
c ========================================[ Problem Statistics ]===========================================
c |                                                                                                       |
c |  Number of variables:         26291                                                                   |
c |  Number of clauses:          100362                                                                   |
c |  Parse time:                   0.03 s                                                                 |
c |                                                                                                       |
c | Preprocesing is fully done
c |  Eliminated clauses:           0.89 Mb                                                                |
c |  Simplification time:          0.11 s                                                                 |
c |                                                                                                       |
c ========================================[ MAGIC CONSTANTS ]==============================================
c | Constants are supposed to work well together :-)                                                      |
c | however, if you find better choices, please let us known...                                           |
c |-------------------------------------------------------------------------------------------------------|
c |                                |                                |                                     |
c | - Restarts:                    | - Reduce Clause DB:            | - Minimize Asserting:               |
c |   * LBD Queue    :     50      |   * First     :   2000         |    * size <  30                     |
c |   * Trail  Queue :   5000      |   * Inc       :    300         |    * lbd  <   6                     |
c |   * K            :   0.80      |   * Special   :   1000         |                                     |
c |   * R            :   1.40      |   * Protected :  (lbd)< 30     |                                     |
c |                                |                                |                                     |
c ==================================[ Search Statistics (every  10000 conflicts) ]=========================
c |                                                                                                       |
c |          RESTARTS           |          ORIGINAL         |              LEARNT              | Progress |
c |       NB   Blocked  Avg Cfc |    Vars  Clauses Literals |   Red   Learnts    LBD2  Removed |          |
c =========================================================================================================
c =========================================================================================================
c restarts              : 6 (350 conflicts in avg)
c blocked restarts      : 0 (multiple: 0) 
c last block at restart : 0
c nb ReduceDB           : 1
c nb removed Clauses    : 997
c nb learnts DL2        : 96
c nb learnts size 2     : 46
c nb learnts size 1     : 11
c conflicts             : 2101           (11360 /sec)
c decisions             : 4101           (0.00 % random) (22174 /sec)
c propagations          : 294438         (1592021 /sec)
c conflict literals     : 28657          (35.45 % deleted)
c nb reduced Clauses    : 16
c CPU time              : 0.184946 s

s UNSATISFIABLE
